content/dstu2/medication-statement.md

```diff
@@ -82,15 +95,18 @@ To successfully POST a MedicationStatement, the following headers must be provid
     x-xss-protection → 1; mode=block
 </pre>
 
-The `etag` response header indicates the current `If-Match` version to use on subsequent updates.
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
 
 ## Update
 
-Update the status of a MedicationStatement, only completed status is supported. The medicationReference field must refer
-to a contained Medication where the code field is populated with the current medication code; it may not be changed.
+Update a MedicationStatement.
 
     PUT /MedicationStatement
 
+_Notes_
+
+* The only change allowed is to change the status to `completed`.
```

Comment:
"allowed" sounds a little authoritative.  Maybe something like "The only supported change is to update the status to `completed`" ?